test/lifecycle/Pausable.test.js

```diff
@@ -64,4 +64,14 @@ contract('Pausable', function () {
     const drasticMeasureTaken = await this.Pausable.drasticMeasureTaken();
     drasticMeasureTaken.should.be.false;
   });
+
+  it('should log Pause and Unpause events appropriately', async function () {
+    const setPauseLogs = (await this.Pausable.pause()).logs;
+    if(setPauseLogs.find(e => e.event === 'Pause') == undefined)
+      assert();
```

Comment:
@elopio  Not sure if this is the right way to do this... However I verified that if the event is not in the set, the test fails as expected.

Reply:
we have tests/helpers/expectEvent.js. It uses assert.exists, which seems better. Here you should use expectEvent. I'll check if this is affecting any of the other tests.